### ISSUE TITLE
we have "Site" twice in the navigation

### ISSUE THREAD

**Issue by OskarStark** [sonata-project/sandbox#566]
<img width="239" alt="bildschirmfoto 2016-01-06 um 10 42 45" src="https://cloud.githubusercontent.com/assets/995707/12139672/44412642-b462-11e5-9933-f6e00c4c4a72.png">

**Comment by davask** [comment]
What about : 

in app/config/sonata/sonata_admin.yml

```yml

[...]

            sonata.admin.group.site_builder:
                label:           Site Builder
                label_catalogue: SonataDemoBundle
                icon:            '<i class="fa fa-puzzle-piece"></i>'
                items:
                    - sonata.page.admin.page
                    - sonata.page.admin.site

            sonata.admin.group.administration:
                label:           sonata_administration
                label_catalogue: SonataAdminBundle
                icon:            '<i class="fa fa-cogs"></i>'
                items:
                    - sonata.user.admin.user
                    - sonata.user.admin.group
                    - sonata.notification.admin.message

[...]

```

**Comment by core23** [comment]
Could you provide a PR @davask ?

**Comment by davask** [comment]
I guess so, but on which branch ?

**Comment by core23** [comment]
Good question...

Ping @sonata-project/contributors

**Comment by davask** [comment]
What do you mean by Ping @sonata-project/contributors ?

**Comment by greg0ire** [comment]
He means he wants comments from member of the @sonata-project/contributors group.

**Comment by OskarStark** [comment]
I recommend `2.4-develop`

**Comment by davask** [comment]
Pull request done

**Comment by OskarStark** [comment]
Closed in #592
